### PR TITLE
initial header set up

### PR DESCRIPTION
**Pull request by ebisLab**
Initial Header set up, no mobile yet.

### CODE REVIEW

File: sass/components/_header.scss [hunk 1]
```diff
@@ -1,15 +1,54 @@
+
 .header {
   &--fixed {
-    position: fixed;
```
Comment (claytron5000) [inline sass/components/_header.scss]: Delete unused code

File: css/styles.css [hunk 2]
```diff
@@ -90,13 +90,17 @@ body {
     background-color: rgba(255, 255, 255, .75);
 }
 
+.header-fixed {
+    background-color: pink;
```
Comment (claytron5000) [inline css/styles.css]: Would we want to use Sass variables for colors?

File: sass/components/_header.scss [hunk 3]
```diff
@@ -1,15 +1,54 @@
...
-    top: 0;
-    left: 0;
-    right: 0;
-    background-color: white;
+
+    // position:relative;
```
Comment (claytron5000) [inline sass/components/_header.scss]: Delete unused code.

File: css/styles.css (hunk 2 above)
Comment (ebisLab) [reply css/styles.css]: no, I was testing this out, I'll remove this asap

**Pull request merged**